### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Justman10000**

### CODE REVIEW

File: components/i18n.tsx [hunk 1]
```diff
@@ -33,11 +77,11 @@ const i18n: { [key: string]: i18nLang } = {
       On Discord, more cool: Gauthier#0875 (or on <a href="https://discord.gg/vCjEufP" target="_blank">my server</a>)
     </>,
     'About page github': () => <>
-      My GitHub, to check my projects: <a href="https://github.com/gauthier-th">https://github.com/gauthier-th</a>
+      My GitHub, check my projects: <a href="https://github.com/gauthier-th">https://github.com/gauthier-th</a>
```
Comment (gauthier-th) [inline components/i18n.tsx:80]: I think the original is ok

File: components/LangButton.tsx [hunk 2]
```diff
@@ -5,6 +5,10 @@ import { ChevronDown } from 'react-feather'
 
 type Langs = { [key: string]: { name: string; icon: JSX.Element } }
 const langs: Langs = {
+  de: {
+    name: 'Deutsch',
+    icon: <img src="https://twemoji.maxcdn.com/svg/1f1e9-1f1ea.svg" className="w-5 h-5" />
+  },
```
Comment (gauthier-th) [inline components/LangButton.tsx:11]: This should be at the end of the `langs` object, so it is at the end of the dropdown

**Review by gauthier-th — CHANGES_REQUESTED** [review]
Thanks for the PR and the german translation!
But I'm curious, why did you make this PR on my personal website?

File: yarn.lock
[the remark this answers is not in the document]
Comment (Justman10000) [reply yarn.lock:1]: The versions are already determined by the package.json?

File: components/i18n.tsx [hunk 3]
```diff
@@ -33,11 +77,11 @@ const i18n: { [key: string]: i18nLang } = {
...
     </>,
     'Index page description': () => <>
       <p>
-        As you might have guessed, my name is Gauthier.
+        As you surely remember, my name is Gauthier.
```
[the remark this answers is not in the document]
Comment (Justman10000) [reply components/i18n.tsx:84]: No, if it is already mentioned above, you do not _think it_, you know it, you have already read it before

File: components/i18n.tsx (hunk 1 above)
Comment (Justman10000) [reply components/i18n.tsx:80]: No, _to check_ is in German, _überprüfen_! So would be:

> Mein GitHub, um meine Projekte zu überprüfen

What sounds stupid! You don't check (or review) projects in this case, you look at them! Is a difference

Comment (Justman10000) [reply components/i18n.tsx:80]: _My GitHub, check my projects_ can be translated in german as

> Mein GitHub, checke meine Projekte

which sounds much better and more suitable

File: components/LangButton.tsx (hunk 2 above)
Comment (Justman10000) [reply components/LangButton.tsx:11]: Why? First comes German, then English, then the other languages! Looks otherwise weird

Comment (Justman10000) [reply components/LangButton.tsx:11]: And I'm not saying this because I'm German

**Comment by Justman10000** [comment]
> But I'm curious, why did you make this PR on my personal website?

I don't quite get your question?

File: yarn.lock
[the remark this answers is not in the document]
Comment (gauthier-th) [reply yarn.lock:1]: Yes, the versions of manually installed libraries are specified in the package.json file, but each library can also have several dependencies, and the versions of these will be calculated and stored in these lock files. This is a good practice that avoids errors when the calculated versions are not the same on different machines. More details [here](https://stackoverflow.com/a/39992365/14306791).

File: components/LangButton.tsx (hunk 2 above)
Comment (gauthier-th) [reply components/LangButton.tsx:11]: I want to prioritize English first, then French, then German - that's the order of priority I want to display.

**Comment by gauthier-th** [comment]
> > But I'm curious, why did you make this PR on my personal website?
> 
> I don't quite get your question?

You've suggested some changes and a German translation, thanks for that, but this is my personal site, no one else is getting any benefit from it. So why suggest these changes?

**Comment by Justman10000** [comment]
> ...no one else is getting any benefit from it.
Isn't that the case with all OpenSource projects?

> So why suggest these changes?
So that even Germans can do something with the page (who can not speak English) 🤣! And to keep all node modules up to date

**Comment by Justman10000** [comment]
> ...no one else is getting any benefit from it.

Isn't that the case with all OpenSource projects?


> So why suggest these changes?

So that even Germans can do something with the page (who can not speak English) 🤣! And to keep all node modules up to date

**Comment by gauthier-th** [comment]
Yeah yeah that's cool, it just seemed unusual to me

File: components/i18n.tsx (hunk 1 above)
Comment (gauthier-th) [reply components/i18n.tsx:80]: Ok

File: components/i18n.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (gauthier-th) [reply components/i18n.tsx:84]: Ok

File: yarn.lock
[the remark this answers is not in the document]
Comment (Justman10000) [reply yarn.lock:1]: Fixed = 360a760

File: components/LangButton.tsx (hunk 2 above)
Comment (Justman10000) [reply components/LangButton.tsx]: Fix = bc80e35

Comment (Justman10000) [reply components/LangButton.tsx]: Fixed = bc80e35

**Review by gauthier-th — CHANGES_REQUESTED** [review]
The space between signs like ! ? are not an error in French, it's the correct way to write it. The correct way of writing "Hey!" is "Salut !" and not "Salut!"

File: components/i18n.tsx [hunk 4]
```diff
@@ -121,11 +77,11 @@ const i18n: { [key: string]: i18nLang } = {
       Sur Discord, plus cool : Gauthier#0875 (ou sur <a href="https://discord.gg/vCjEufP" target="_blank">mon serveur</a>)
     </>,
     'About page github': () => <>
-      Mon GitHub, pour mes projets : <a href="https://github.com/gauthier-th">https://github.com/gauthier-th</a>
```
Comment (gauthier-th) [inline components/i18n.tsx:80]: I prefer the original, it sounds weird in French.

File: components/i18n.tsx [hunk 5]
```diff
@@ -121,11 +77,11 @@ const i18n: { [key: string]: i18nLang } = {
...
+      Mon GitHub, vérifie mes projets : <a href="https://github.com/gauthier-th">https://github.com/gauthier-th</a>
     </>,
     'Index page description': () => <>
       <p>
-        Comme vous avez pu le deviner, je m'appelle Gauthier.
+        Comme tu t'en souviens sûrement (si tu as été sur la page à propos de moi), je m'appelle Gauthier.
```
Comment (gauthier-th) [inline components/i18n.tsx:84]: Same, it sounds weird in French.

**Review by gauthier-th — COMMENTED** [review]
The space between signs like ! ? are not an error in French, it's the correct way to write it. The correct way of writing "Hey!" is "Salut !" and not "Salut!"

File: components/i18n.tsx
Comment (gauthier-th) [inline components/i18n.tsx:1]: The space between signs like ! ? are not an error in French, it's the correct way to write it. The correct way of writing "Hey!" is "Salut !" and not "Salut!"

**Pull request closed without merging**